docs/en/administration/fault_diagnosis_and_analysis.md

```diff
@@ -129,17 +131,20 @@ You need to add the `juicefs.access-log` configuration item in the [client confi
 By default, JuiceFS clients will listen to a TCP port locally via [pprof](https://pkg.go.dev/net/http/pprof) to get runtime information such as Goroutine stack information, CPU performance statistics, memory allocation statistics. You can see the specific port number that the current JuiceFS client is listening on by using the system command (e.g. `lsof`):
 
 :::note
-If JuiceFS is mounted via the root user, then you need to add `sudo` before the `lsof` command.
+If you mount JuiceFS as the root user, you need to add `sudo` before the `lsof` command.
```

Comment:
```suggestion
If JuiceFS is mounted with the root user, you need to add `sudo` before the `lsof` command.
```
Try to avoid first-person pronouns. :)

Reply:
Would it be better to replace `with` with `by`?